tsconfig.json

```diff
@@ -20,6 +20,8 @@
       "es2019",
       "esnext"
     ],
+    "resolveJsonModule": true,
```

Comment:
@petemill do you think this is okay? I did it so I could get strong typings for the localization message keys.

Reply:
I think so. Nice idea.